components/storage/blockdevice/COMPONENT_SD/config/mbed_lib.json

```diff
@@ -8,7 +8,8 @@
         "FSFAT_SDCARD_INSTALLED": 1,
         "CMD_TIMEOUT": 10000,
         "CMD0_IDLE_STATE_RETRIES": 5,
-        "SD_INIT_FREQUENCY": 100000
+        "INIT_FREQUENCY": 100000,
```

Comment:
Why did the the name change?

Reply:
Initial name setting was a typo, "SD_" was not needed as the tools append "MBED_CONF_SD_" for all the config options, with old name the macro would be "MBED_CONF_SD_SD_INIT_FREQUENCY"